tools/src/main/java/org/apache/kafka/trogdor/workload/ConsumeBenchSpec.java

```diff
@@ -61,6 +62,12 @@
  * #{@link org.apache.kafka.clients.consumer.KafkaConsumer#subscribe(Collection)}.
  * It will be assigned partitions dynamically from the consumer group.
  *
+ * This specification supports the spawning of multiple consumers in the single Trogdor worker agent.
+ * The "threadCount" field denotes how many consumers should be spawned for this spec.
+ * It is worth noting that the "targetMessagesPerSec", "maxMessages" and "activeTopics" fields apply for every consumer individually.
+ * If a consumer group is not specified, every consumer is assigned a different, random group. When specified, all consumers use the same group.
+ * Specifying partitions, a consumer group and multiple consumers will result in an #{@link ConfigException} and the task will abort.
+ *
```

Comment:
It would probably be good to have a separate paragraph to describe the group configuration.

We probably want to mention that if individual partitions are specified, only a single consumer is supported, and subscribe is not used.